### ISSUE TITLE
[FR] Skip the first X fields in multiplelabels printing.

### ISSUE THREAD

**Issue by roman-dvorak** [inventree/InvenTree#4929]
When you are printing labels on a sheet of stickers, it's useful to have the option to skip the first few labels. When I need to print only, let's say, 5 labels out of 40 (whole sheet), it results in large waste. If I can't reuse the rest of the sheet. So it would be great if the label generation process allows skipping the first x labels, and use rest of the sheet.

_Originally posted by @roman-dvorak in https://github.com/inventree/InvenTree/issues/4805#issuecomment-1568537132_

I haven't seen the current implementation of issue #4805 (I will review it after it's merged and probably I will expand this issue). When the page with labels is generating, I would like to be able to select the layout type and specify the number of labels to skip.

Best to solve after #4805 and #4926.

**Comment by SchrodingersGat** [comment]
@roman-dvorak good idea - we'll consider this after the referenced issues are merged

**Comment by martonmiklos** [comment]
I think for implementing this properly we need to have some user defined settings UI presented at/after the label type selection. I have to study @wolflu05  's machine integration PR to see if that covers such a feature.

**Comment by wolflu05** [comment]
Not really. Currently the machine integration is just a registry that stored machines. Each machine can define a type and their user provided driver. The current label printing plugins need to be rewritten in a driver. A driver is as simple as a class that provides a print_labels function like the current plugins too.

**Comment by SchrodingersGat** [comment]
I think that we should let each "printing machine" specify custom options which are presented to the user when printing is initiated

**Comment by martonmiklos** [comment]
> I think that we should let each "printing machine" specify custom options which are presented to the user when printing is initiated

Well it looks to be a new issue/PR for me to discuss.

This paginated printing stuff looks to be a rabbit hole which exposes moore rabbit holes on the bottom for me :D

**Comment by wolflu05** [comment]
This can later be implemented with printing options that are also required for the machine integration.

**Comment by SchrodingersGat** [comment]
not stale

**Comment by wolflu05** [comment]
Shouldn't that be added to the built-in plugin?
